include/util/debug_geometry.hpp

```diff
@@ -77,12 +81,25 @@ inline void DEBUG_GEOMETRY_START(const contractor::ContractorConfig &config)
     timeinfo = localtime(&raw_time);
     strftime(dg_time_buffer, 80, "%Y-%m-%d %H:%M %Z", timeinfo);
 
+    boost::filesystem::ifstream nodes_input_stream{config.node_based_graph_path, std::ios_base::in|std::ios_base::binary};
+
+    extractor::QueryNode current_node;
+    unsigned number_of_coordinates = 0;
+    nodes_input_stream.read((char *)&number_of_coordinates, sizeof(unsigned));
+
+    for (unsigned i = 0; i < number_of_coordinates; ++i)
+    {
+        nodes_input_stream.read((char *)&current_node, sizeof(extractor::QueryNode));
+        node_lookup_map[current_node.node_id] = util::FixedPointCoordinate(current_node.lat, current_node.lon);
+    } nodes_input_stream.close();
```

Comment:
Run `clang-format` (similat to gofmt for example) that re-formats against our `.clang-format` specs.